Gem/Code/Source/Automation/ScriptManager.cpp

```diff
@@ -1164,9 +1164,9 @@ namespace AtomSampleViewer
 
     void ScriptManager::Script_CapturePassAttachment(AZ::ScriptDataContext& dc)
     {
-        if (dc.GetNumArguments() != 3)
+        if (dc.GetNumArguments() != 3 && dc.GetNumArguments() != 4)
```

Comment:
Nitpick, standard Lua is to allow extra arguments and just not do anything with them. So this should likely be written to only fuss if there are less than 3 arguments.